src/kili/services/export/format/llm/__init__.py

```diff
@@ -0,0 +1,143 @@
+        with output_filename.open("wb") as output_file:
+            output_file.write(export_json.encode("utf-8"))
+
+    def process_and_save(self, assets: List[Dict], output_filename: Path) -> None:
+        """LLM specific process and save."""
+        result = []
+        for asset in assets:
+            jobs_config = self.project["jsonInterface"]["jobs"]
+            json_response = asset["latestLabel"]["jsonResponse"]
+            result.append(
+                {
+                    "raw_data": _format_raw_data(asset),
+                    "status": asset["status"],
+                    "external_id": asset["externalId"],
+                    "metadata": asset["jsonMetadata"],
+                    "labels": [_format_json_response(jobs_config, json_response)],
+                }
+            )
+
+        self._save_assets_export(result, output_filename)
+
+
+def _format_json_response(
+    jobs_config: Dict, json_response: Dict
+) -> Dict[str, Union[str, List[str]]]:
+    result = {}
+    for job_name, job_value in json_response.items():
+        job_config = jobs_config[job_name]
+        if job_config is None:
+            continue
+        if job_config["mlTask"] == "CLASSIFICATION":
+            result[job_name] = []
+            for category in job_value["categories"]:
+                result[job_name].append(category["name"])
+                if "children" in category:
+                    for child_name, child_value in _format_json_response(
+                        jobs_config, category["children"]
+                    ).items():
+                        result[f"{job_name}.{category['name']}.{child_name}"] = child_value
+        elif job_config["mlTask"] == "TRANSCRIPTION":
+            result[job_name] = job_value["text"]
+        else:
+            logging.warning(f"Job {job_name} with mlTask {job_config['mlTask']} not supported")
+    return result
+
+
+def _format_raw_data(asset) -> List[Dict]:
+    raw_data = []
+
+    chat_id = asset["jsonMetadata"].get("chat_id", None)
+
+    if (
+        "chat_item_ids" in asset["jsonMetadata"]
+        and isinstance(asset["jsonMetadata"]["chat_item_ids"], str)
+        and len(asset["jsonMetadata"]["chat_item_ids"]) > 0
+    ):
+        chat_items_ids = str.split(asset["jsonMetadata"]["chat_item_ids"], "_")
+    else:
+        chat_items_ids = []
+
+    if (
+        "models" in asset["jsonMetadata"]
+        and isinstance(asset["jsonMetadata"]["models"], str)
+        and len(asset["jsonMetadata"]["models"]) > 0
+    ):
+        models = str.split(asset["jsonMetadata"]["models"], "_")
+    else:
+        models = []
+
+    with Path(asset["content"]).open(encoding="utf8") as file:
+        data = json.load(file)
+    version = data.get("version", None)
+    if version == "0.1":
+        for index, prompt in enumerate(data["prompts"]):
+            raw_data.append(
+                {
+                    "role": prompt.get("title", None),
```

Comment:
There is no key "title" on the prompt object unfortunately.
Maybe we can have a default value of "user" in this case ? 
See discussion on slack.